learntools/embeddings/nbs/3-gensim.ipynb

```diff
@@ -383,9 +245,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "#### DB: Is 2.7 large or small? It'd be interesting to compare it to the Euclidien distance between two randomly chosen movies.\n",
```

Comment:
Does cell 9 below where we look at the euclidean/cosine distance between *Toy Story* and *The Exorcist* address this concern?

Reply:
todo: rearrange to bring resolution closer